### ISSUE TITLE
Source datasets should be listed somewhere

### ISSUE THREAD

**Issue by qgroom** [trias-project/risk-maps#40]
The source datasets should be listed somewhere. I assume this will be the same ones as for the cube.
Ultimately a dynamic sources citation would be best, but a list will do for now.

**Comment by niconoe** [comment]
For the occurrence data, could the cube be listed as a source? If so, wouldn't that indirectly references all primary sources?

**Comment by peterdesmet** [comment]
Yeah, I think we can cite the cube: https://doi.org/10.5281/zenodo.4299976. The cube itself cites the two GBIF downloads.

**Comment by qgroom** [comment]
Very indirectly. I don't see where the cube is citing the occurrence data at all!

**Comment by damianooldoni** [comment]
![image](https://user-images.githubusercontent.com/33662631/117443464-77ed4580-af38-11eb-993c-a549be09c751.png)

in _Related identifiers_.

**Comment by qgroom** [comment]
Wow! Who would know to look there!
I don't think that really gives credit to the data providers.
Couldn't the risk maps harvest that DOI and make the link on the website? Preferably actually listing the top 10 contributors.
You might think I'm being picky, but I get a lot of blowback from scientist at the Garden and from citizen scientists that they are not given sufficient recognition.

**Comment by niconoe** [comment]
Yeah, I understand the concerns about recognition!

I never tried, but I guess by following DOIs and using Zenodo's API we could retrieve the indirect contributions as structured data, to be presented on the website. That would be ideal, because:

- Fully dynamic, so always up-to-date (if the raw data sources changes) and without the risk of forgetting someone
- Could be presented quite visibly on the website for good recognition

I'll investigate that approach!

**Comment by peterdesmet** [comment]
@niconoe @qgroom it might not be super visible to a human, but the good thing is that at least it's machine readable: https://zenodo.org/record/4299976/export/dcite4#.YJU6MWYza88:

```
<relatedIdentifier relatedIdentifierType="DOI" relationType="IsDerivedFrom" resourceTypeGeneral="Dataset">10.15468/dl.cvumnw</relatedIdentifier>
```
